examples/settings/Kconfig

```diff
@@ -0,0 +1,53 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+config EXAMPLES_SETTINGS
+	tristate "Settings example"
+	default n
+	---help---
+        Enable the "settings" example. This uses the functions provided in
+        utils/settings to test the storage and retrieval od various
```

Comment:
```suggestion
        utils/settings to test the storage and retrieval of various
```

Reply:
Found some spaces not tabs in this too, so will correct that and this error, and push a new commit